covin_slot_tracker.py

```diff
@@ -183,7 +183,10 @@ def TelegramBot(TOKEN, CHAT_ID, MSG):
                 if available:
                     msg_body = "Slots Available at {total} places.\n{available}".format(total = total_centers,available = available)
                     print(msg_body)
-                    TelegramBot(SECRET_TOKEN,CHAT_ID,msg_body)
+                    telegram_txt = str((f'Slots Available at {total_centers} places.\n{available}'))
+                    base_url = 'https://api.telegram.org/bot1801976621:AAHk9BvV4r9d_ITf-wCppIqcGYNQQMsK5cs/sendMessage?chat_id={}&text={}'.format(Telegram_ID,
```

Comment:
Use python-telegram-bot to connect to the bot and not have the secret token exposed in the code